ports/color-console/vcpkg.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "color-console",
+  "version-semver": "1.0.0",
```

```suggestion
  "version-date": "2022-03-20",
```
The value of the version-date should be the date you submitted this PR.